### PR TITLE
List using Go

### PR DESCRIPTION
**Pull request by msk397**

### CODE REVIEW

File: codes/go/chapter_array_and_linkedlist/list_test.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	/* 访问元素 */
+	num := list[1] // 访问索引 1 处的元素
+	fmt.Println("访问索引 1 处的元素，得到 num =", num)
+
+	/* 更新元素 */
+	list[1] = 0 // 将索引 1 处的元素更新为 0
+	fmt.Println("将索引 1 处的元素更新为 0 ，得到 list =", list)
+
+	/* 清空列表 */
+	list = []int{}
```
Comment (krahets) [inline codes/go/chapter_array_and_linkedlist/list_test.go:28]: I think `list = []int{}` is to initialize but not clear a slice. It'd be better to use `list = nil` instead,

**Review by krahets — COMMENTED** [review]
Thanks @msk397 ! Please address the comment.

File: codes/go/chapter_array_and_linkedlist/list_test.go (hunk 1 above)
Comment (msk397) [reply codes/go/chapter_array_and_linkedlist/list_test.go]: You are right, I have finished modifying!

**Review by krahets — APPROVED** [review]
Thanks for the work! Now merge it.

**Pull request merged**